addons/core/translations/en-us.yaml

```diff
@@ -40,7 +40,6 @@ titles:
   feature-edition: 'Edition: {edition}'
 editions:
```

Comment:
Do we still need this translation? Seems like we don't use this anywhere in the code

Reply:
Removed.